### PR TITLE
Bump detect-secrets version

### PR DESCRIPTION
**Pull request by jasper-vandemalle**
Same as #21, but with the version change.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 repos:
   - repo: .
-    rev: v0.7.3
+    rev: v0.7.5
```
Comment (jasper-vandemalle) [inline .pre-commit-config.yaml:4]: I guess this can't be 0.7.5 yet, until after we tag.

Comment (jasper-vandemalle) [inline .pre-commit-config.yaml:4]:
```suggestion
    rev: v0.7.4
```

**Comment by jasper-vandemalle** [comment]
Kinda wondering if #21 is a better solution 🤔

**Comment by ls-alexander-muehlbauer** [comment]
> Kinda wondering if #21 is a better solution 🤔

I'm thinking either one would be fine. Whichever we merge first 😆 
Since we already missed to do a few bumps for the version the tag in the pyproject.toml is incorrect now anyway and would be correct after this PR + tag

**Review by ls-alexander-muehlbauer — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (ls-alexander-muehlbauer) [reply .pre-commit-config.yaml]: Maybe it could? Is it even needed if it's pointing to itself?

Comment (jasper-vandemalle) [reply .pre-commit-config.yaml]: I tried in https://app.circleci.com/pipelines/github/lightspeed-hospitality/pre-commit-hooks/144/workflows/775441a0-5e70-40d8-95bd-9a94472ce6aa/jobs/123

**Pull request merged**